docs/source/about/contract.rst

```diff
@@ -63,64 +91,86 @@ MUST EITHER:
-* The Makefile SHOULD delegate non-trivial work to helper scripts written in an appropriate language
-   * Helper scripts MAY live in paasta/ and MAY be named after the Makefile targets that call them, e.g. paasta/test, paasta/itest,  etc.
-* MUST respond to make test
-* SHOULD respond to ``make test`` by running unit tests
-* MUST respond to ``make itest`` by generating a Docker image which will run with the currently checked-out code
-* SHOULD respond by running some kind of system-level test (integration, e2e, contract, smoke) which exercises the generated Docker image
-* MUST accept a parameter $(DOCKER_TAG) to make itest. Jenkins will calculate a tag for the newly-built Docker image and pass it to the service’s build scripts via this parameter::
+       # $(DOCKER_TAG) SHOULD default to something sensible when the job is run outside of Jenkins (e.g. during local development).
+       # E.g.:
+       DOCKER_TAG ?= <servicename>-dev-$(USER)
+       ...
+       build:
+       docker build -t $(DOCKER_TAG) .
 
-    $(DOCKER_TAG) SHOULD default to something sensible when the job is run outside of Jenkins (e.g. during local development).
-    E.g.:
-    DOCKER_TAG ?= <servicename>-dev-$(USER)
-    ...
-    build:
-    docker build -t $(DOCKER_TAG) .
+* The Makefile SHOULD respond to ``make cook-image`` by generating a Docker image tagged with ``$(DOCKER_TAG)`` for use by local-run.
 
-* SHOULD respond to make cook-image by generating a Docker image tagged with $(DOCKER_TAG) for use by local-run
-* cook-image MAY be added as a dependency of the itest target so that both Jenkins and local-run use the same image creation process::
+   * cook-image MAY be added as a dependency of the itest target so that both Jenkins and local-run use the same image creation process::
 
-    itest: cook-image
-        ...
-    cook-image:
-        docker build -t "$$DOCKER_TAG" .
+       itest: cook-image
+           ...
+       cook-image:
+           docker build -t "$$DOCKER_TAG" .
 
-* yelpsoa-configs/<your service> MUST contain a deploy.yaml and SHOULD use jenkins.yaml to create Jenkins pipelines based on that deploy.yaml.
+* ``yelpsoa-configs/<your service>`` MUST contain a ``deploy.yaml``.
+  See `y/deploy-yaml <https://y.yelpcorp.com/deploy-yaml>`_ for more information about deploy.yaml.
+
+* your service's git repo SHOULD contain a ``jenkins.yaml`` to create Jenkins pipelines based on that deploy.yaml.
+  See `y/jenkinsfile <https://y.yelpcorp.com/jenkinsfile>`_ for more info.
 
 Manual Steps (Not Recommended)
 ------------------------------
 
-This section describes how to operate the PaaSTA deploy system. If for some
-reason you don’t want to use Jenkins, you can pull the levers yourself. The
-PaaSTA CLI contains subcommands to help with this (these same subcommands are
-used by Jenkins)
+This section describes how to operate the PaaSTA deploy system.
+If for some reason you don’t want to use Jenkins, you can pull the levers yourself.
```

Comment:
i feel like we should have stronger wording here about how unrecommended this is - but that's just me :p